api/client.ml

```diff
@@ -62,7 +62,13 @@ end
 
 module Org = struct
   type t = Raw.Client.Org.t Capability.t
-  type repo_info = { name : string; master_status : Build_status.t }
+
+  type repo_info = {
+    name : string;
+    main_status : Build_status.t;
+    main_hash : string;
```

Comment:
There are some type aliases around for `git_hash` and the like. 
It would be worthwhile to make their use consistent throughout this file.